applications/openshift/etcd/etcd_unique_ca/oval/shared.xml

```diff
@@ -24,7 +24,13 @@
   </ind:textfilecontent54_object>
 
   <ind:textfilecontent54_state id="state_etcd_unique_ca" version="1">
-    <ind:subexpression datatype="string" operation="pattern match">^/etc/etcd/ca.crt$</ind:subexpression>
+    <ind:subexpression datatype="string" operation="pattern match">^/etc(|/ssl)/etcd/ca.crt$</ind:subexpression>
```

Comment:
If OCP3 is being evaluated, is ``/etc/ssl/etcd/ca.crt`` acceptable, or does OCP3 expect ``/etc/etcd/ca.crt``?

Reply:
OCP3 sets up the CA as `/etc/etcd/ca.crt`, so changing the path and such I don't think makes sense.